lib/topic_query.rb

```diff
@@ -165,7 +165,8 @@ def list_latest
 
   def list_read
     create_list(:read, unordered: true) do |topics|
-      topics.order('COALESCE(tu.last_visited_at, topics.bumped_at) DESC')
+      topics.where('tu.last_visited_at IS NOT NULL')
+        .order('COALESCE(tu.last_visited_at, topics.bumped_at) DESC')
```

Comment:
But I can see the change in result. Normally it includes new topics. After i added the `where` condition it showing correctly.

Anyway adding spec will give best solution for this 👍